src/esp/gfx/PbrShader.cpp

```diff
@@ -119,10 +119,8 @@ PbrShader::PbrShader(Flags originalFlags, unsigned int lightCount)
                      ? "#define NONE_ROUGHNESS_METALLIC_TEXTURE\n"
                      : "")
       .addSource(flags_ & Flag::NormalTexture ? "#define NORMAL_TEXTURE\n" : "")
-      .addSource(flags_ & Flag::NormalTextureScale
-                     ? "#define NORMAL_TEXTURE_SCALE\n"
-                     : "")
       .addSource(flags_ & Flag::ObjectId ? "#define OBJECT_ID\n" : "")
+      .addSource(flags_ & Flag::ClearCoatLayer ? "#define CLEAR_COAT\n" : "")
```

Comment:
Oops one flag got past me as I cherry picked this. If this antagonizes anyone :grin: I'll get rid of it, but it's going to be added in the next PR anyway.